apps/react-18-tests-v9/webpack.serve.config.js

```diff
@@ -0,0 +1,21 @@
+const resources = require('../../scripts/webpack/webpack-resources');
+const getResolveAlias = require('../../scripts/webpack/getResolveAlias');
```

Comment:
wondering how is this working for v9 as this is legacy v8 setup.

we can make this extremely fast and use v9 DX:

```js
const path = require('path');

const HtmlWebpackPlugin = require('html-webpack-plugin');
const { TsconfigPathsPlugin } = require('tsconfig-paths-webpack-plugin');

module.exports = () => {
  return {
    entry: ['react-app-polyfill/ie11', './src/index.tsx'],
    output: {
      filename: 'react-18-tests-v9.js',
      path: path.resolve(__dirname, 'dist'),
    },
    resolve: {
      extensions: ['.tsx', '.ts', '.js'],
      plugins: [
        new TsconfigPathsPlugin({
          configFile: path.resolve(__dirname, '../../tsconfig.base.json'),
        }),
      ],
      alias: {
        react: path.resolve(__dirname, './node_modules/react'),
        'react-dom': path.resolve(__dirname, './node_modules/react-dom'),
      },
    },
    module: {
      rules: [
        {
          test: /\.tsx?$/,
          exclude: /node_modules/,
          use: 'swc-loader',
        },
      ],
    },
    plugins: [
      new HtmlWebpackPlugin({
        templateContent: '<div id="root"></div>',
      }),
    ],
  };
};

```

**with this you can remove**

- `config/pre-copy.json`
- `index.html`
- use just `webpack.config.js`

**Note:** 
- you'll need to install `swc-loader` or wait for this PR to get in https://github.com/microsoft/fluentui/pull/24279/files
- with this new config you need to update your package.json#scripts
```diff
{
-    "build": "just-scripts build",
    "lint": "just-scripts lint",
    "test": "just-scripts test",
    "just": "just-scripts",
    "clean": "just-scripts clean",
    "code-style": "just-scripts code-style",
+  "type-check": "tsc -b",
-    "start": "just-scripts dev"
+ "start": "webpack-dev-server --mode=development --config=webpack.serve.config.js",
}
```